configure.ac

```diff
@@ -4301,6 +4301,14 @@ AC_MSG_RESULT([$ntp_ok])
 
 ###
 
+if test "$SOURCE_DATE_EPOCH"; then
+    DATE_FMT="%d %b %Y %H:%M:%S"
```

Comment:
This is not portable, e.g. on FreeBSD it will produce `date: illegal option -- d`